serving/installing-logging-metrics-traces.md

```diff
@@ -1,83 +1,76 @@
-   reported `Running` or `Completed`:
+Knative offers three different setups for collecting logs:
 
-     ```shell
-     kubectl get pods --namespace monitoring --watch
-     ```
+1. [Elasticsearch and Kibana](#elasticsearch-and-kibana)
+1. [Stackdriver](#stackdriver)
+1. [Custom logging plugin](setting-up-a-logging-plugin.md)
 
-     ```
-     NAME                                  READY     STATUS    RESTARTS   AGE
-     elasticsearch-logging-0               1/1       Running   0          2d
-     elasticsearch-logging-1               1/1       Running   0          2d
-     fluentd-ds-5kc85                      1/1       Running   0          2d
-     fluentd-ds-vhrcq                      1/1       Running   0          2d
-     fluentd-ds-xghk9                      1/1       Running   0          2d
-     grafana-798cf569ff-v4q74              1/1       Running   0          2d
-     kibana-logging-7d474fbb45-6qb8x       1/1       Running   0          2d
-     kube-state-metrics-75bd4f5b8b-8t2h2   4/4       Running   0          2d
-     node-exporter-cr6bh                   2/2       Running   0          2d
-     node-exporter-mf6k7                   2/2       Running   0          2d
-     node-exporter-rhzr7                   2/2       Running   0          2d
-     prometheus-system-0                   1/1       Running   0          2d
-     prometheus-system-1                   1/1       Running   0          2d
-     ```
+### Elasticsearch and Kibana
+
+Run:
 
-  CTRL+C to exit watch.
+```shell
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.2/monitoring-logs-elasticsearch.yaml
+```
 
-### Create Elasticsearch Indices
+The installation is complete when logging & monitoring components are all
+reported `Running` or `Completed`:
 
-To visualize logs with Kibana, you need to set which Elasticsearch indices to explore. We will
-create two indices in Elasticsearch using `Logstash` for application logs and `Zipkin`
-for request traces.
+```shell
+kubectl get pods --namespace monitoring --watch
+```
+
+```text
+NAME                                  READY     STATUS    RESTARTS   AGE
+elasticsearch-logging-0               1/1       Running   0          2d
+elasticsearch-logging-1               1/1       Running   0          2d
+fluentd-ds-5kc85                      1/1       Running   0          2d
+fluentd-ds-vhrcq                      1/1       Running   0          2d
+fluentd-ds-xghk9                      1/1       Running   0          2d
+kibana-logging-7d474fbb45-6qb8x       1/1       Running   0          2d
+```
+
+Hit CTRL+C to exit watch and continue to [Create Elasticsearch Indices](#create-elasticsearch-indices) section.
+
+#### Create Elasticsearch Indices
+
+To visualize logs with Kibana, you need to set which Elasticsearch indices to explore.
 
 - To open the Kibana UI (the visualization tool for [Elasticsearch](https://info.elastic.co)),
   you must start a local proxy by running the following command:
```

Comment:
I can't comment far enough down, but there's a `/monitoring/` URL below which should be `/knative-monitoring/`

Reply:
fixed